services/jobs/src/main/java/com/linkedin/openhouse/jobs/model/JobConf.java

```diff
@@ -17,6 +19,7 @@
 public class JobConf {
   private JobType jobType;
   private String proxyUser;
+  @Builder.Default private Map<String, String> executionConf = new HashMap<>();
```

Comment:
Do we want to keep this super narrow and store vars executor.memory and driver.memory, or allow arbitrary Spark properties?

Reply:
I preferred NOT to allow arbitrary spark props to control and have consistency in our maintenance jobs. Arbitrary spark props can lead to job failures and the toil to identify and fix them would be big.
Also, in terms of UX, since memory is only table level spark  property, I prefer to keep it specific.